src/records/a_record.rs

```diff
@@ -171,23 +39,56 @@ impl<'a> ARecord<'a> {
         if line.len() < 7 {
             return Err(ParseError::SyntaxError);
         }
+
+        // check for old spec format (e.g. `AC00069`)
+        //
+        // all known loggers that use this old format use numeric serial numbers
+        // so we assume that to be a sufficient heuristic to detect this format
+        if line.bytes().skip(2).take(5).all(|b| b.is_ascii_digit()) {
+            if !line.as_bytes()[1].is_ascii() {
```

Comment:
`line[0..1] == "A"` already on line 37

Reply:
yes, but this is `line[1]`, not `line[0]`